libs/scripts/scripts.js

```diff
@@ -36,9 +36,11 @@ const stageDomainsMap = {
   },
   '.business-graybox.adobe.com': { 'business.adobe.com': 'origin' },
   // TODO: remove after QE approval
-  '--milo--robert-bogos.hlx.page': {
-    'www.adobe.com': 'main--dc--adobecom.hlx.page',
-    'business.adobe.com': 'business.stage.adobe.com',
+  '^https://.*--milo--robert-bogos.hlx.page': {
```

Comment:
Should this be `*--milo--*.hlx.page` so that it works on every page?

More generally, I'd love us to have great example settings as consumers often default to just copying it over from somewhere.

Reply:
Totally agree. I've made the change for this to work on every page 👍